symfony/client/src/orangehrmPimPlugin/components/SaveLicense.vue

```diff
@@ -61,6 +61,7 @@
               v-model="license.expiryDate"
               :rules="rules.expiryDate"
               type="date"
+              :years="endYear"
```

Comment:
`endYear` also not make sense for year array

Reply:
use yearArray instead